README.adoc

```diff
@@ -16,42 +16,46 @@
+Explore how to use Minishift to deploy microservices to an Origin Community Distribution of Kubernetes (OKD) cluster. 
 
 // *********************************************************************************************
 
 == What you'll learn 
 
-You will learn how to deploy two simple microservices with Open Liberty to an OKD cluster running in Minishift.
+You will learn how to deploy two simple microservices with Open Liberty to an Origin Community Distribution of Kubernetes (OKD) cluster that is running in 
+Minishift.
 
-OKD, formerly known as OpenShift Origin, is the upsteam open-source project for all OpenShift products. OKD is a Kubernetes-based platform with added
-functionality. If you would like to learn more about Kubernetes, check out the 
-https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^] guide. OKD streamlines the DevOps process by providing an 
-intuitive development pipeline. It also provides integration with multiple tools to make the deployment and management of cloud applications easier. To 
-learn more about the different platforms that Red Hat OpenShift offers, check out the https://docs.openshift.com[official OpenShift documentation^]. To 
-learn how to deploy microservices to OpenShift Online, check out the https://openliberty.io/guides/cloud-openshift.html[Deploying microservices to OpenShift^] 
-guide.
+=== What is Origin Community Distribution of Kubernetes (OKD)?
 
-Using Maven, you will build a simple microservice called `system` that collects basic system properties from your system. Then, you will build another 
-microservice, `inventory` that will interact with the `system` microservice, and learn how to deploy both to the cluster and establish communication between 
-them.
+OKD, formerly known as OpenShift Origin, is the upstream open source project for all OpenShift products. OKD is a Kubernetes-based platform with
+added functionality. OKD streamlines the DevOps process by providing an intuitive development pipeline. It also provides integration with multiple tools 
+to make the deployment and management of cloud applications easier.
 
-You will use Minishift, an all-in-one virtual machine built on top of Minikube. Similar to how Minikube makes it easy to deploy a local Kubernetes cluster 
-locally, Minishift allows a quick and easy environment for application development that supports all OKD features.
+To learn more about the different platforms that Red Hat OpenShift offers, check out the 
+https://docs.openshift.com[official OpenShift documentation^]. To learn how to deploy microservices to OpenShift Online, check out the 
+https://openliberty.io/guides/cloud-openshift.html[Deploying microservices to OpenShift^] guide. If you would like to learn more about Kubernetes, check out 
+the https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^] guide.
```

Comment:
We should update the links above to point to OKD instead?  Not sure we need links to OpenShift?  We can keep the link to the Kubernetes guide.